### ISSUE TITLE
Add Zombie retrospective kind

### ISSUE THREAD

**Issue by coralieco** [hchevalier/retrospective#285]
We could adapt the "Retro dont vous êtes le héro" as "Walking dead", "Zombie" or another-title-in-english.

It requires a bit a drawing. 

Here is the full explanations => https://blog.octo.com/la-retrospective-dont-vous-etes-le-heros/

**Comment by hchevalier** [comment]
YES! This is the one I called "Day-Z" in the readme, and this is why retrospectives already have a `sub_step` column that is not used yet 😅

Feel free to contribute to this one 👍
